src/pykeen/trackers/base.py

```diff
@@ -36,3 +38,72 @@ def end_run(self) -> None:
 
         HAS to be called after the experiment is finished.
         """
+
+
+class ConsoleResultTracker(ResultTracker):
+    """A class that directly prints to console."""
+
+    def __init__(
+        self,
+        *,
+        print_parameters: bool = True,
+        parameter_filter: Union[None, str, Pattern[str]] = None,
+        print_metrics: bool = True,
+        metric_filter: Union[None, str, Pattern[str]] = None,
+        start_end_run: bool = False,
+    ):
+        """
+        Initialize the tracker.
+
+        :param print_parameters:
+            Whether to print parameters.
+        :param parameter_filter:
+            A regular expression to filter parameters. If None, print all parameters.
+        :param print_metrics:
+            Whether to print metrics.
+        :param metric_filter:
+            A regular expression to filter metrics. If None, print all parameters.
+        :param start_end_run:
+            Whether to print start/end run messages.
+        """
+        self.start_end_run = start_end_run
+
+        self.print_parameters = print_parameters
+        if isinstance(parameter_filter, str):
+            parameter_filter = re.compile(parameter_filter)
+        self.parameter_filter = parameter_filter
+
+        self.print_metrics = print_metrics
+        if isinstance(metric_filter, str):
+            metric_filter = re.compile(metric_filter)
+        self.metric_filter = metric_filter
+
+    def start_run(self, run_name: Optional[str] = None) -> None:  # noqa: D102
+        if run_name is not None and self.start_end_run:
+            print(f"Starting run: {run_name}")
```

Comment:
better to use `tqdm.write` throughout this PR than print so it displays properly if there are progress bars

Reply:
Does this one also work if we do not have an active tqdm bar?

Reply:
done in 074cb69